src/declarative.js

```diff
@@ -134,15 +134,24 @@
       //    By disaling this module write-protection we gain performance.
       //    It could be useful to allow an option to enable or disable this.
       module.locked = true;
-      moduleObj[name] = value;
+
+      // export({name: value})
+      if (typeof name == 'object') {
+        for (var p in name)
```

Comment:
Only if we expect to run in environments with Object.prototype extended with enumerable properties.